cpp/inc/bond/ext/grpc/unary_call_result.h

```diff
@@ -24,14 +23,9 @@ namespace bond { namespace ext { namespace gRPC {
         ///
         /// @param response The response.
         /// @param status The status.
-        /// @param context the context under which the request is being executed.
-        unary_call_result(
-            bonded<Response> response,
-            const grpc::Status& status,
-            std::shared_ptr<grpc::ClientContext> context)
```

Comment:
If we remove the context from here, then clients won't be able to get access to the server's metadata unless they explicitly create a context themselves.